src/services/LiveSubmissionsService.js

```diff
@@ -4,17 +4,24 @@ import VerdictIssuedEvent from "../models/VerdictIssuedEvent";
 
 export class LiveSubmissionsService extends AbstractService {
 
-    constructor(studentService, problemService, stompClient) {
+    constructor(studentService, problemService, submissionService, stompClient) {
         super({
             baseURL: process.env.REACT_APP_PROBLEM_SVC_BASE_URL,
```

Comment:
Should it be `REACT_APP_SUBMISSION_SVC_BASE_URL` like you did in the `src/services/SubmissionService.js`?

Reply:
Great catch